### ISSUE TITLE
--out throws error if folder does't exist

### ISSUE THREAD

**Issue by SkipHendriks** [vega/ts-json-schema-generator#994]
I'm currently using this library to create schemas for request validation in our CI. At the point of generating the first schema the `/schemas` folder doesn't yet exist. I could of course add a `touch -p schema` line to the script, but it would feel more straightforward to have it created automatically, possibly behind a flag. I think this would improve the experience using this tool in a CI environment.

The error thrown:
```
[Error: ENOENT: no such file or directory, open './schemas/SomeSchema.json'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: './schemas/SomeSchema.json'
}
```

I'd be happy to provide a PR if the maintainers think this is useful as well.

**Comment by domoritz** [comment]
Sounds good. Please send a pull request.

**Comment by SkipHendriks** [comment]
Fixed by #1000
